addons/binding/org.openhab.binding.somfytahoma/ESH-INF/thing/exteriorscreen.xml

```diff
@@ -9,7 +9,6 @@
 		</supported-bridge-type-refs>
 		<label>Somfy Exterior Screen</label>
 		<channels>
-			<channel id="position" typeId="position"></channel>
```

Comment:
Is this channel supported / used in the past, does this installations, should we mention it in release notes?

Reply:
This is a relic from the former review, the implementation has been removed but channel not.